src/macaron/slsa_analyzer/analyzer.py

```diff
@@ -31,9 +31,6 @@
     PURLNotFoundError,
     RepoCheckOutError,
 )
-from macaron.malware_analyzer.checks import (  # pylint: disable=unused-import # noqa: F401
-    detect_malicious_metadata_check,
-)
 from macaron.output_reporter.reporter import FileReporter
```

Comment:
Is this import removed because it's not used within this file ?

Reply:
Yes.

Reply:
Oh right, pylint was disabled for this statement, I was wondering why the pre-commit hooks didn't mark it as error.

Reply:
Look like this line was critical for the `detect_malicious_metadata_check` to run :sweat_smile: .
Apparently, the registering of `detect_malicious_metadata_check.py` here https://github.com/oracle/macaron/blob/739826e0ff285f452e66abc9b222c77008d853fc/src/macaron/malware_analyzer/checks/detect_malicious_metadata_check.py#L296 doesn't work because `detect_malicious_metadata_check.py` is not located in `src/macaron/slsa_analyzer/checks` where all checks modules are loaded here https://github.com/oracle/macaron/blob/84728190397faf40571b14a5a822e47c8fd7cdc7/src/macaron/slsa_analyzer/checks/__init__.py#L11
This also explains why an explicit import line for `detect_malicious_metadata_check` is put in this file. I guess we can consider move that `detect_malicious_metadata_check` module into its correct location. 
cc: @behnazh-w .

Reply:
Thanks @tromai for looking into this. Yes, we need to move the check under `src/macaron/slsa_analyzer/checks`. @benmss Please revert this change and I will fix it separately.